q2_feature_table/tests/test_filter.py

```diff
@@ -203,6 +203,44 @@ def test_sample_metadata(self):
         expected = Table(np.array([]), [], [])
         self.assertEqual(actual, expected)
 
+        # exclude one
```

Comment:
Sorry @maxvonhippel, I missed another one here: exclude none. This would looks like metadata with index values that aren't found in the table.